addons/mail/static/src/components/composer/composer.js

```diff
@@ -300,6 +313,21 @@ class Composer extends Component {
         this._postMessage();
     }
 
+    _onFocusinSubject() {
+        this.composer.update({ subjectContent: this._subjectRef.el.value });
+    }
+
+    _onFocusoutSubject() {
+        this.composer.update({ subjectContent: this._subjectRef.el.value });
+    }
+
+    _onKeyupSubject() {
+        this.composer.update({ subjectContent: this._subjectRef.el.value });
+    }
+
+    _onInputSubject() {
+        this.composer.update({ subjectContent: this._subjectRef.el.value });
+    }
```

Comment:
Can we just double check the events that we actually need in current state. Isn't just input enough?

Reply:
PS: care to what we did for mentions where we actually have "all the other events" but not input, to avoid the thing being laggy when typing fast/keeping key pressed down.

Reply:
Input seems indeed enough